azurerm_app_insights/main.tf

```diff
@@ -3,6 +3,10 @@
 data "azurerm_resource_group" "rg" {
   name = "${local.azurerm_resource_group_name}"
 }
+data "azurerm_key_vault" "key_vault" {
```

Comment:
Add a blank line between the two data blocks

Reply:
done